api/src/repositories/survey-block-repository.ts

```diff
@@ -19,7 +19,7 @@ export const SurveyBlockRecord = z.object({
   survey_id: z.number(),
   name: z.string(),
   description: z.string(),
-  geojson: z.any(),
+  geojson: z.any().nullable(),
```

Comment:
praise: Technically this nullable is not needed (any will override) but I think this is good for readability